game/src/js/Game.js

```diff
@@ -13,7 +13,14 @@ class Game extends Phaser.Game {
       pixelArt: true,
       width: 800,
       height: 600,
-      scene: [LoaderScene, TitleScene, DemoScene, DungeonScene, PauseMenuScene, DialogScene]
+      scene: [LoaderScene, TitleScene, DemoScene, DungeonScene, PauseMenuScene, DialogScene],
+      physics: {
+        default: 'impact',
```

Comment:
I think it's better if we use the default `arcade` physics engine. The `impact` engine is meant for compatibility with ImpactJS, but we are not porting anything over from it.